changelog/Microsoft.PIM.AzureRBAC.json

```
@@ -1,5 +1,223 @@
+          "Target": "unifiedRoleManagementAlertConfiguration"
+        },
+        {
+          "Id": "7ed39a6d-65eb-4b0d-8de0-1c1a6897347a",
+          "ApiChange": "Resource",
+          "ChangedApiName": "unifiedRoleManagementAlertDefinition",
+          "ChangeType": "Addition",
+          "Description": "Added the [unifiedRoleManagementAlertDefinition](https://learn.microsoft.com/en-us/graph/api/resources/unifiedRoleManagementAlertDefinition?view=graph-rest-beta) resource",
+          "Target": "unifiedRoleManagementAlertDefinition"
+        },
+        {
+          "Id": "7ed39a6d-65eb-4b0d-8de0-1c1a6897347a",
+          "ApiChange": "Resource",
+          "ChangedApiName": "unifiedRoleManagementAlertIncident",
+          "ChangeType": "Addition",
+          "Description": "Added the [unifiedRoleManagementAlertIncident](https://learn.microsoft.com/en-us/graph/api/resources/unifiedRoleManagementAlertIncident?view=graph-rest-beta) resource",
+          "Target": "unifiedRoleManagementAlertIncident"
+        },
+        {
+          "Id": "7ed39a6d-65eb-4b0d-8de0-1c1a6897347a",
+          "ApiChange": "Method",
+          "ChangedApiName": "refresh",
+          "ChangeType": "Addition",
+          "Description": "Added the [refresh](https://learn.microsoft.com/en-us/graph/api/unifiedRoleManagementAlert?view=graph-rest-beta) method to the [unifiedRoleManagementAlert](https://learn.microsoft.com/en-us/graph/api/resources/unifiedRoleManagementAlert?view=graph-rest-beta) resource.",
+          "Target": "unifiedRoleManagementAlert"
+        },
+        {
+          "Id": "7ed39a6d-65eb-4b0d-8de0-1c1a6897347a",
+          "ApiChange": "Method",
+          "ChangedApiName": "refresh",
+          "ChangeType": "Addition",
+          "Description": "Added the [refresh](https://learn.microsoft.com/en-us/graph/api/unifiedRoleManagementAlert?view=graph-rest-beta) method to the [unifiedRoleManagementAlert](https://learn.microsoft.com/en-us/graph/api/resources/unifiedRoleManagementAlert?view=graph-rest-beta) resource.",
+          "Target": "unifiedRoleManagementAlert"
+        },
+        {
+          "Id": "7ed39a6d-65eb-4b0d-8de0-1c1a6897347a",
+          "ApiChange": "Method",
+          "ChangedApiName": "remediate",
+          "ChangeType": "Addition",
+          "Description": "Added the [remediate](https://learn.microsoft.com/en-us/graph/api/unifiedRoleManagementAlertIncident?view=graph-rest-beta) method to the [unifiedRoleManagementAlertIncident](https://learn.microsoft.com/en-us/graph/api/resources/unifiedRoleManagementAlertIncident?view=graph-rest-beta) resource.",
+          "Target": "unifiedRoleManagementAlertIncident"
+        }
+      ],
+      "Id": "7ed39a6d-65eb-4b0d-8de0-1c1a6897347a",
+      "Cloud": "Prod",
+      "Version": "beta",
+      "CreatedDateTime": "2023-01-31T17:44:49.9324302Z",
+      "WorkloadArea": "Identity and access",
+      "SubArea": "Identity governance"
```

To map to the docs and changelog taxonomy
```suggestion
      "SubArea": "Governance"
```